Sources/Support/PaywallExtensions.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PaywallExtensions.swift
+//
+//  Created by Nacho Soto on 6/6/23.
+
+import StoreKit
+import SwiftUI
+
+#if swift(>=5.9)
+
+@available(iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0, *)
+extension StoreView {
+
+    /// Creates a view to load a collection of products from the App Store, and merchandise them.
+    /// When the user purchases products through this paywall, the `RevenueCat` SDK will handle
+    /// the result automatically. All you need to do is to dismiss the paywall
+    public init(
+        offering: Offering,
+        prefersPromotionalIcon: Bool = false
+    ) where Icon == EmptyView, PlaceholderIcon == EmptyView {
+        self.init(
+            ids: offering.allProductIdentifiers,
```

Comment:
I wonder... would these products be ordered by the iOS UI? Just it would be great to avoid showing the yearly price above the monthly for example. 

If it doesn't and it depends on the order of the product ids passed maybe we should try to make that customizable.

Reply:
Oh good point! In fact I noticed the ordering wasn't stable. I'll sort by duration 👍🏻

Reply:
I think that works for a V1. For the future, it would be great to allow devs to customize the order (so we don't limit what they can do with the iOS widget), but that's probably not trivial, so I think we can leave that for later

Reply:
They can always use the paywall API directly in that case :)

Reply:
Sorting done 👍🏻